portfolio/src/main/webapp/style.css

```diff
@@ -11,10 +11,11 @@ body {
 }
 
 #fullpage {
-  min-width: 100vw;
+min-width: 100vw;
```

Comment:
indent and maintain sorted order

Reply:
Indented here 17b27e0 and Alphabetised here e27539f